tests/cypress/support/commands.js

```diff
@@ -1704,3 +1700,15 @@ Cypress.Commands.overwrite('reload', (orig, options) => {
     orig(options);
     cy.closeModalUnsupportedPlatform();
 });
+
+Cypress.Commands.add('clickDeleteFrame', (force = false) => {
+    cy.get('.cvat-player-delete-frame').click();
+    cy.get('.cvat-modal-delete-frame').within(() => {
+        cy.contains('button', 'Delete').click({ force });
+    });
+});
+
+Cypress.Commands.add('clickSave', (force = false) => {
```

Comment:
I will suggest being more specific here in naming if we add global Cypress command.
We probably have many `Save` buttons across the application. 

So, maybe `clickSaveAnnotationView`.
Additionally I will ask to search by `contains('Save').click` over the tests and replace findings

I found at least three more places with this code:
```
cy.get('button').contains('Save').click();
cy.get('button').contains('Save').trigger('mouseout');
```